richardaspinall/poker-react-node
# feat: table ready logic

### Description

Backend logic to check whether a table is full and thus ready to start a game.

### How to test

1. npm run start:dev
2. Click on one seat
3. Open second browser
4. Click on the other seat
5. Check console logs for "starting game"

### Task

- https://app.clickup.com/t/86cu6paeu
- CU-86cu6paeu
